### DISCUSSION TITLE
Concatenate N vectors

### DISCUSSION THREAD

**Discussion by Tommaso11397** [stan-dev/stanc3#925] [Q&A] [answered]
Hi, I have N vectors v_n of dimension K x 1, I would like to concatenate all these vectors into a a unique vector of dimension N*K.
Obviously I cannot use append_row, I tried with for loop but I could not do it. Any idea? Thanks in advance.

**Comment by wds15** [comment, marked as answer]
Sorry, I meant stan discourse forums.

**Reply by Tommaso11397** [reply]
Now you can answer the question here :)

https://discourse.mc-stan.org/t/concatenating-n-vectors/23433?u=tommaso11397
